src/API/Query.php

```diff
@@ -582,6 +585,41 @@ public function setWildcardSearchEnabled($wildcardSearchEnabled)
     }
 
     /**
+     * <b>Warning</b>  This will count as two queries against your search index.
+     *
+     * Typically, this feature is used when you have a large number of navigation items that will overwhelm the end
+     * user. It works by using one of the existing navigation items to decide what the query is about and fires a second
+     * query to restrict the navigation to the most relevant set of navigation items for this search term.
+     *
+     * For example, if you pass in a search of `paper` and a restrict navigation of `category:2`
+     *
+     * The bridge will find the category navigation refinements in the first query and fire a second query for the top 2
+     * most populous categories.  Therefore, a search for something generic like "paper" will bring back top category
+     * matches like copy paper (1,030), paper pads (567).  The bridge will fire off the second query with the search
+     * term, plus an OR refinement with the most likely categories.  The navigation items in the first query are
+     * entirely replaced with the navigation items in the second query, except for the navigation that was used for the
+     * restriction so that users still have the ability to navigate by all category types.
+     *
+     * JSON Reference:
+     *
+     *     { "restrictNavigation": { "navigationName": "category", "refinementCount": 2 }
+     *
+     * @param $restrictNavigation Restriction criteria
+     */
+    public function setRestrictNavigation($restrictNavigation)
+    {
+        $this->restrictNavigation = $restrictNavigation;
+    }
+
+    /**
+     * @return restrictNavigation Restriction criteria
```

Comment:
Return tag should only reference the return type

/** @return RestrictNavigation */